packages/backend/src/storage/storage.ts

```diff
@@ -98,6 +100,7 @@ export class Storage extends EventEmitter {
       // @ts-ignore
       AccessControllers: AccessControllers
     })
+    this.emit(SocketActionTypes.TOR_CONNECTION_PROCESS, TorConnectionProcessInfo.INITIALIZED_STORAGE)
```

Comment:
I don't think it's right that we have storage specific events under TorConnectionProcessInfo => I can imagine possible scenarios that it misleads someone in the future.